### ISSUE TITLE
Fails to Load on 2021.8.0 published version

### ISSUE THREAD

**Issue by mlohus93** [garbled1/homeassistant_ecowitt#70]
Getting "(!) Not Loaded" for the Ecowitt Integration.  Worked flawlessly for me prior to installing HA 2101.8.0.

Logs:
2021-08-04 09:29:39 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration ecowitt which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-08-04 09:29:39 ERROR (SyncWorker_1) [homeassistant.loader] The custom integration 'ecowitt' does not have a valid version key (None) in the manifest file and was blocked from loading. See https://developers.home-assistant.io/blog/2021/01/29/custom-integration-changes#versions for more details
2021-08-04 09:29:43 ERROR (MainThread) [homeassistant.setup] Setup failed for ecowitt: Integration not found.

**Comment by riddler2608** [comment]
We had this issue a while ago you need to add version no in and then you will have a issue about "volt" fix that and it should be all good ! Look through older issues and it tells you how to fix both issues.

**Comment by DaDre** [comment]
Issue 68 resolved it for me. Renamed all VOLT to ELECTRIC_POTENTIAL_VOLT in const.py file

**Comment by Steve2017** [comment]
Made the changes in Issue 68 and it leaves me with the same error. Where it refers to "<" as the error, is that a proxy for something else?

I had done the version change before.

And just as I am typing this a version **_core-2021.8.2_** has just dropped. I will try that.

**Comment by gameover-nl** [comment]
> Issue 68 resolved it for me. Renamed all VOLT to ELECTRIC_POTENTIAL_VOLT in const.py file

This also fixed it for me in the 2021.8.2 version. :)

**Comment by Steve2017** [comment]
I might have spoken too soon. Out of interest I tried a restart..... same error comes up.

Ecowitt might have to go.

**Comment by gameover-nl** [comment]
> I might have spoken too soon. Out of interest I tried a restart..... same error comes up.
> 
> Ecowitt might have to go.

Hmm, I made the changes and did a restart (shutdown host) and everything booted fine without errors.

**Comment by BubiBalboa** [comment]
It works but you have to replace every instance of VOLT and change nothing else.  Easy to make a mistake here.

**Comment by gameover-nl** [comment]
I used search & replace, watching out for not changing the instances of "VOLTAGE" in stead of "VOLT".

**Comment by mr-sneezy** [comment]
I fixed both update issues too BTW, I did not need to uninstall & reload the Ecowitt Integration, just edited const.py and manifest.json and restarted HA server.

**Comment by Steve2017** [comment]
> I fixed both update issues too BTW, I did not need to uninstall & reload the Ecowitt Integration, just edited const.py and manifest.json and restarted HA server.

Which version did you enter in manifest.json? I originally had "1.0.0" but then changed it for the reinstall to what it says on GitHub - "0.5.0".
Also in that file I notice in the fix suggested by @jonootto he increases the value here from 0.11 to 0.12:
```
"requirements": [
    "pyecowitt==0.12"

```

**Comment by mr-sneezy** [comment]
The top half of my manifest.json as is right now...

{
  "domain": "ecowitt",
  "name": "Ecowitt Weather Station",
  "version": "7.5.1",
  "config_flow": true,
  "documentation": "https://www.home-assistant.io/integrations/ecowitt",
  "requirements": [
    "pyecowitt==0.11"

**Comment by Steve2017** [comment]
Thanks

I've been following something on the "official" GitHub and it looks like there are bigger issues effecting sensors. (eg: Picks up the Envoy solar PV system) There is another update in the works - hopefully in the next 24 hours.

**Comment by mr-sneezy** [comment]
Is that something to do with Statistical Graphs in Lovelace ?
I tried to add indoor & outdoor temp, but the entities are not listed. 

A note links me to:
https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics

**Comment by Steve2017** [comment]
https://github.com/home-assistant/core/pull/54142

based on:
https://github.com/home-assistant/core/issues/54104#issuecomment-894202280

**Comment by Steve2017** [comment]
And core-2021.8.3 is out

**Comment by muster021** [comment]
In case someone, like me, wants to add the integration for the first time and it does not show up in the Integrations list, no matter if you add version number and edit const.py, add it to your configuration.yaml:
```
ecowitt:
  port: 4199
```
 This will make the integration show up under configured integrations.

**Comment by Liranan79** [comment]
> In case someone, like me, wants to add the integration for the first time and it does not show up in the Integrations list, no matter if you add version number and edit const.py, add it to your configuration.yaml:
> 
> ```
> ecowitt:
>   port: 4199
> ```
> 
> This will make the integration show up under configured integrations.

Thanks Muster,
Adding a version number and changing VOLT to ELECTRIC_POTENTIAL_VOLT was not enough for me. Adding ecowitt  in the configuration.yaml fixed it for me in 2021.8.3

**Comment by mlohus93** [comment]
Thank you everyone...
Issues 65 and 68 did help but I had similar situation as Steve2017 had where once I rebooted I lost all my Ecowitt devices. What I figured out was that I already had a version entry ("version": "7.5.1",) in my manifest.json file and I added "version" : "0.5.0", towards the top of the file as shown in an early post to that issue, resulting in me having two version entries.  Took out the original version 7.5.1 string and following a reboot, Ecowitt works fine.   Since I had a version string already, it is possible I didnt need to add or replace mine, but happy to have it work whatever version number is referenced.

**Comment by mlohus93** [comment]
Thank you everyone...
Issues 65 and 68 did help but I had similar situation as Steve2017 had where once I rebooted I lost all my Ecowitt devices. What I figured out was that I already had a version entry ("version": "7.5.1",) in my manifest.json file and I added "version" : "0.5.0", towards the top of the file as shown in an early post to that issue, resulting in me having two version entries. Took out the original version 7.5.1 string and following a reboot, Ecowitt works fine. Since I had a version string already, it is possible I didnt need to add or replace mine, but happy to have it work whatever version number is referenced.
